### PR TITLE
chore(interactive-examples): add border radius

### PR DESCRIPTION
**Pull request by ElijahPepe**
Before:
https://i.elijahpepe.com/v8zUE90.png

After:
https://i.elijahpepe.com/c3bIXNF.png

### CODE REVIEW

**Comment by schalkneethling** [comment]
Thank you for your pull request @ElijahPepe. Changes to the design and layout of MDN Web Docs need to go through an internal triage process before we can consider implementing changes. This is to ensure that the website follows Mozilla’s overall design principles and meets the brand guidelines.

As such, we cannot immediately review and merge your pull request but, we will take this suggestion under advisement. In general, it is best to start off a suggestion or change proposal via a GitHub issue. That way we can discuss the proposal and once we come to an agreement, it can be implemented and integrated via the pull request workflow. I hope this process is not too cumbersome.

Thank you again for using and contributing to MDN Web Docs.

**Comment by caugner** [comment]
@ElijahPepe Could you add "Before" and "After" screenshots to the PR description to make it easier for us to understand the impact of this PR? Thanks!

**Comment by ElijahPepe** [comment]
> @ElijahPepe Could you add "Before" and "After" screenshots to the PR description to make it easier for us to understand the impact of this PR? Thanks!

Will do, thanks!

**Comment by caugner** [comment]
> Will do, thanks!

I don't see any difference between the screenshots you added.

FYI The easiest way to add a screenshot here on GitHub is to paste them in the editor from your clipboard (I use this most of the time, to avoid local files), by drag'n'dropping the image file into the editor, or by clicking the "Attach files ..." line below the editor.

**Comment by ElijahPepe** [comment]
> > Will do, thanks!
> 
> @ElijahPepe I'm afraid I don't see any difference between the screenshots you added.

I will admit the difference is very subtle between the two images, which isn't true on the web. If you examine the pixels in the edges of the before and after pictures you can see them easier.

Before:
xTTrJif

After:
WBWT2Kl

**Comment by schalkneethling** [comment]
Thank you for your pull request @ElijahPepe. I believe the larger question here is why you think this is a good change. Do you feel it adds more consistency with the rest of the design or, is there another reason. I am not opposed it, just wanted to understand your reasoning for the change. Thanks!

**Comment by ElijahPepe** [comment]
@schalkneethling Well, I don't really have a larger motive besides every other element on MDN, like tips and the table, have a border-radius of 0.25rem so it was a bit inconsistent when the interactive examples didn't.

**Comment by caugner** [comment]
@ElijahPepe Thanks, this is some very important context to understand the goal of this PR.

Please make sure to always provide this context in the PR description upfront. 🙏 

Here are some elements that ✅ have a `border-radius` or :x: not:

✅  Callouts:
image

✅  Code blocks:
image

✅  Browser compat table:
image

✅ Notecards:
image

:x: Markdown tables:
image

:x: HTML tables:
image

File: client/src/document/interactive-examples.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   background-color: var(--background-secondary);
   border: 1px solid var(--border-primary);
   color: var(--text-primary);
+  border-radius: var(--elem-radius);
```
Comment (caugner) [inline client/src/document/interactive-examples.scss:7]: Small nit:

```suggestion
  border-radius: var(--elem-radius);
  color: var(--text-primary);
```

**Review by caugner — CHANGES_REQUESTED** [review]

**Comment by schalkneethling** [comment]
> @schalkneethling Well, I don't really have a larger motive besides every other element on MDN, like tips and the table, have a border-radius of 0.25rem so it was a bit inconsistent when the interactive examples didn't.

That is what I thought might be the reason. Good enough reason for me, thank you for your contribution @ElijahPepe 🎉

**Review by schalkneethling — APPROVED** [review]

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
Thanks @ElijahPepe!

**Pull request merged**